pkg/network/namescheme/networknamescheme.go

```diff
@@ -46,11 +52,11 @@ func CreateNetworkNameScheme(vmiNetworks []v1.Network) map[string]string {
 func mapMultusNonDefaultNetworksToPodInterfaceName(networks []v1.Network) map[string]string {
 	networkNameSchemeMap := map[string]string{}
 	for i, network := range vmispec.FilterMultusNonDefaultNetworks(networks) {
-		networkNameSchemeMap[network.Name] = secondaryInterfaceName(i + 1)
+		networkNameSchemeMap[network.Name] = SecondaryInterfaceName(i + 1)
 	}
 	return networkNameSchemeMap
 }
 
-func secondaryInterfaceName(idx int) string {
+func SecondaryInterfaceName(idx int) string {
```

Comment:
Why now this method should be public? I couldn't find any place in this commit using it.